services/system/Transact/include/services/system/RTransact.hpp

```diff
@@ -83,17 +83,27 @@ namespace SystemService
       std::vector<TraceClientInfo> clients;
    };
    PSIO_REFLECT(TraceClientRow, id, clients)
-
    using TraceClientTable = psibase::Table<TraceClientRow, &TraceClientRow::id>;
 
+   // Holds the size of the last transaction trace
```

Comment:
I was expecting just a printf, which will show up in the psinode log (`debug` on success `warning` on failure). This query would not usable on a public endpoint because of the race.

Reply:
The log on a successful callback is debug level, so it isn't shown by default. It's too noisy to be on by default.

Reply:
I adjusted my filter to `Severity >= info or Channel = transaction` to see all the callbacks.

Reply:
Done, and you're right - tested, printf works if I adjust logger